src/main/java/net/sf/jabref/importer/EntryFromPDFCreator.java

```diff
@@ -68,12 +67,12 @@ public boolean accept(File f) {
         return Optional.of(res.getEntries().get(0));
 
         /*addEntryDataFromPDDocumentInformation(pdfFile, entry);
-        addEntyDataFromXMP(pdfFile, entry);
-
+        addEntryDataFromXMP(pdfFile, entry);
+        
         if (entry.getField("title") == null) {
         	entry.setField("title", pdfFile.getName());
         }
-
+        
         return entry;*/
```

Comment:
Maybe. However, in that case we can also remove addEntryDataFromPDDocumentInformation and  addEntryDataFromXMP (if I recall correctly) and it seems like those could be useful if they are used. Of course we can recreate those if/when needed.